bitnami/rabbitmq-cluster-operator/values.yaml

```diff
@@ -127,6 +127,12 @@ clusterOperator:
     ##
     pullSecrets: []
 
+  ## @param clusterOperator.watchAllNamespaces Watch for resources in all namespaces
+  ##
+  watchAllNamespaces: true
+  ## @param clusterOperator.watchNamespaces [array] Watch for resources in the given namespaces
+  ##
+  watchNamespaces: []
```

Comment:
I would add a comment stating that `watchNamespaces` is ignored if `watchAllNamespaces` is `true`. This may sound obvious, but one could interpret "watch all namespaces in this list" 🙂